src/magicbox/MagicBox.ts

```diff
@@ -146,7 +146,7 @@ export class MagicBoxInstance {
       }
       if (key == 13) {
         // Enter
-        const suggestion = this.suggestionsManager.select();
+        const suggestion = this.suggestionsManager.getCurrentlyFocusedWithKeyboardElement();
```

Comment:
Could switch the words around for a more concise name: `getKeyboardFocusedElement`.